.gitignore

```diff
@@ -46,7 +46,7 @@ yarn-error.log
 .lighthouseci
 .direnv/*
 .envrc
-
+.python-version
```

Comment:
@PranshuSrivastava Is this an expected change? (Also, looks like this is added twice)

Note: We already have a separate PR to add this file in the gitignore! :)

Reply:
Hey @DubeySandeep, I had fixed this in the last commit. Even in my local repo, it's not there.